resources/templates/md/posts/2018-09-10-sept-meetup.md

```diff
@@ -0,0 +1,36 @@
+{:title "The hack-session meetup on Sept 1, 2018"
+ :layout :post
+ :tags  ["Clojure" "ClojureScript"]
+ :author "Amarjeet"}
+
+
+We hosted the 2nd episode of the hack-session during this [Sept meetup](https://www.meetup.com/Bangalore-Clojure-User-Group/events/253965766/) at Nilenso office. 1st episode was during the [Aug meetup](https://www.meetup.com/Bangalore-Clojure-User-Group/events/252993263/).
+
+Not only we had larger participation, Clojurians could also push their PRs to open-sourced projects. And, some are in production already 🙂
```

Comment:
Consider dropping "And," as the first word of the last sentence. Also, put a full stop after the last sentence.

Reply:
Fixed.